utils/autogenerate_all_imports.py

```diff
@@ -124,13 +157,20 @@ def write_inits(folder_path: str, all_dict: dict, skip_children: Dict[str, List[
         in_docstring = in_end_content = False
         in_start_content = True
         for line in path.read_text(encoding='utf8').splitlines():
-            if not in_start_content and line.startswith('# END AUTO-GENERATED'):
+            if not in_start_content and line.startswith(f'# END {AUTO_GENERATED}'):
                 in_end_content = True
             if in_end_content:
                 end_content.append(line)
                 continue
-            # only use initial comments and docstring
-            if not (line.startswith('#') or line.startswith('"""') or in_docstring):
+            # Only keep initial comments and docstring
+            # TODO: Support __future__ imports, for which we will also
```

Comment:
I originally tried to support `__future__` imports in `__init__` files but it proved to be tricky. Also, once we drop 3.9 support, `from __future__ import annotations` should be less important since we'll have the `|` union operator finally.